modules/javafx.controls/src/test/java/test/javafx/scene/control/TreeTableViewTest.java

```diff
@@ -2591,14 +2636,14 @@ public void updateItem(String item, boolean empty) {
 
         StageLoader sl = new StageLoader(treeTableView);
 
-        assertEquals(22, rt_31200_count);
+        assertEquals(18, rt_31200_count);
```

Comment:
magic number... should we explain why exactly 18 ?

Reply:
It is the count, how often update item was called. 
That it got smaller shows, that we call it less often, as the item was not changed (which is a good thing).
That also matches with your observation, which is the expected output here.

I saw that this was changed for some tests in https://github.com/openjdk/jfx/pull/863, we also can do something similar here as well.

Reply:
on second thought, this is ok as is - I asked because the code is not testing a specific requirement.
The second `assertEquals(18)` might be testing a specific requirement - that the count has not changed, right?

Reply:
Yes, as far as I can see.

Reply:
so in theory, we could just test the fact that the count has not changed:
```
int ct = rt_31200_count;
...
assertEquals(ct, rt_31200_count);
```

this will make the test independent of the first number (which might change) and only test what's important (that the count does not change later).

what do you think?